src/index.js

```diff
@@ -2,34 +2,36 @@ import peg from 'pegjs';
 
 import sanToTextGrammar from './grammar/sanToText';
 import textToSanGrammar from './grammar/textToSan';
+import sanToTextGrammarES from './grammar/sanToTextES';
+import textToSanGrammarES from './grammar/textToSanES';
 
 // Rules that can be configured by the user. For example, users could add
 // aliases for common misspellings of terms, like "night" instead of "knight"
 const configurableRules = {
     // Pieces
     king: {
         name: 'king',
-        defaultTerms: ['king'],
+        defaultTerms: ['king', 'rey'],
```

Comment:
Now I see that the way I did the aliases for pieces makes it a little tricky to add multiple languages. This works, but it's not ideal. I'll try to think of a way that doesn't mix the different languages in the same grammar.

Reply:
I thought about this more and one possibility would be to move the default terms and aliases into a separate config file, e.g. aliases.json:

```
{
    "en": {
        "king": {
            "defaultTerms": ["king"],
            ...
        },
        ...
        "one": {
            "defaultTerms": ["1", "one"]
            ...
        },
        ...
    },
    "es": {
        "king": {
            "defaultTerms": ["rey"],
            ...
        },
        ...
        "one": {
            "defaultTerms": ["1", "uno"]
            ...
        },
        ...
    },
    ...
}
```

It could be done with a single config file, or with a separate config file per language, e.g. with a directory structure like this:

```
+ grammar
    + en
        + sanToText
        + textToSan
        + aliases.json
    + es
        + sanToText
        + textToSan
        + aliases.json
    ...
```

Reply:
Agree! I like separate config file per language solution.